<?xml version="1.0"?>
<launch>
<group ns="drone03">
  <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
  <arg name="name" default="drone03"/>
  <arg name="tf_prefix" value="drone03"/>
  <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor_downward_cam.gazebo.xacro"/>
<!--   <arg name="x" default="0.0"/>
  <arg name="y" default="0.0"/>
  <arg name="z" default="0.2"/> -->
  <arg name="x" default="2.0"/>
  <arg name="y" default="2.0"/>
  <arg name="z" default="0.3"/>
  </include>
  </group>
  <node pkg="rosservice" type="rosservice" name="enable_motors_d3" args="call --wait /drone03/enable_motors true"/>
</launch>